### DISCUSSION TITLE
taro是如何保障多个小程序的一致性

### DISCUSSION THREAD

**Discussion by happywuming** [NervJS/taro#14937] [Q & A]
### 这个特性解决了什么问题？
taro是否有对应的自动化测试工具，用来保障多个小程序的一致性，能减少测试对于不同小程序的验证

### 这个 API 长什么样？
自动化测试工具

**Comment by ZakaryCode** [comment]
参考[文档](https://nervjs.github.io/taro-docs/docs/test-utils/)
